src/System.Net.Http/tests/UnitTests/HttpSystemProxyTest.cs

```diff
@@ -112,7 +132,31 @@ public void HttpProxy_Local_Parsing(string bypass, int count)
                 }
                 return SuccessExitCode;
            }, bypass, count.ToString()).Dispose();
+        }
+
+        [InlineData("http://")]
+        [InlineData("http=")]
+        [InlineData("http://;")]
+        [InlineData("http=;")]
+        [InlineData("  ;  ")]
+        [InlineData("proxy.contoso.com")]
+        public void HttpProxy_InvalidSystemProxy_Null(string proxyString)
+        {
+            IWebProxy p;
+
+            FakeRegistry.Reset();
```

Comment:
Similar to other unit tests that are using the "FakeRegistry", this test might need to be enclosed in a RemoteInvoke to isolate it for running as a separate process. Otherwise, parallel execution of these tests will fail randomly since FakeRegistry is a static thing.

Reply:
I'm running into some marshaling issues, that may require me to rework things a bit, but I should be able to make this change.